protocol-specs/generators/block_processing_transfers/index.js

```diff
@@ -2132,106 +90,142 @@ const accounts = {
 		balance: '0',
 		delegateName: 'genesis_100',
 	},
+	newAccount: {
+		passphrase:
+			'blame address tube insect cost knock major level regret bring april stick',
+		privateKey:
+			'b92e223981770c716ee54192a0ad028639d28d41221b72e455447bc4767aeb94caff2242b740a733daa3f3f96fc1592303b60c1704a8ac626e2704da039f41ee',
+		publicKey:
+			'caff2242b740a733daa3f3f96fc1592303b60c1704a8ac626e2704da039f41ee',
+		address: '2222471382442610527L',
+		balance: '0',
+	},
+	newAccount2: {
```

Comment:
Not sure if `newAccount2` is the best naming here? 😄

Reply:
Changed names